Sources/Utility/ArgumentParser.swift

```diff
@@ -200,6 +203,26 @@ public struct PathArgument: ArgumentKind {
     public static var completion: ShellCompletion = .filename
 }
 
+public struct ExecutableName: ArgumentKind {
+    public let name: String
+
+    public init(argument: String) throws {
+        name = argument
+    }
+
+    public static var completion: ShellCompletion = .function("_swift_executable")
```

Comment:
The two new structs are doing same thing as a StringArgument, so we should probably find a way to use that. Maybe we can make `ShellCompletion` an instance variable instead of static?